packages/editor-ui/src/components/ContactPromptModal.vue

```diff
@@ -100,7 +99,7 @@ export default mixins(workflowHelpers).extend({
 						type: 'success',
 					});
 				}
-				this.$store.commit('ui/closeTopModal');
+				this.modalBus.$emit('close');
```

Comment:
refactor to use modal bus event instead

Reply:
I remember this from one of the past PR. Like :)